pilot/pkg/bootstrap/configcontroller.go

```diff
@@ -141,6 +141,7 @@ func (s *Server) initK8SConfigStore(args *PilotArgs) error {
 			return err
 		}
 	}
+	s.RWConfigStore = configController
```

Comment:
I am not sure this is the right abstraction here. Its only applied to k8s config store - other stores support writing as well, although they are admittedly less used. Second, aggregate config controller (s.controller) already supports a MakeWriteableCache which allows it to handle write actions. Would it make sense to use that instead of adding a new field? Just worried about confusion from ~3 fields that are all about "config stores"